exporter/trace/cloudtrace_test.go

```diff
@@ -154,14 +90,16 @@ func TestExporter_DisplayNameFormatter(t *testing.T) {
 
 	// wait exporter to flush
 	flush()
-	assert.EqualValues(t, 1, mockTrace.len())
-	assert.EqualValues(t, "TEST_FORMAT"+spanName, mockTrace.spansUploaded[0].DisplayName.Value)
+	assert.EqualValues(t, 1, mock.GetNumSpans())
+	assert.EqualValues(t, "TEST_FORMAT"+spanName, mock.GetSpan(0).DisplayName.Value)
 }
 
 func TestExporter_Timeout(t *testing.T) {
 	// Initial test precondition
-	mockTrace.spansUploaded = nil
-	mockTrace.delay = 20 * time.Millisecond
+	mock := cloudmock.NewCloudMock()
+	defer mock.Shutdown()
+	mock.SetDelay(20 * time.Millisecond)
```

Comment:
This will also be impossible to set in other languages.
Ideally there should be a Docker image with flags to make this functionality portable to other languages.

Reply:
As mentioned above, we were thinking of doing something like [this](https://medium.com/learning-the-go-programming-language/calling-go-functions-from-other-languages-4c7d8bcc69bf) to allow other languages to use our `cloudmock` library. We could also turn `SetDelay` and other functions into gRPC endpoints on our server so that other languages can still call it. We hadn't really thought of creating a Docker image, but it's definitely something we can consider.

Reply:
I would think having endpoints or command flags to set configuration would be a lot simpler than calling into Go code from another language. From you comment above it doesn't look like you are going that way, as I don't think you can both "spin up the server separately" and "call into Go code" (for a separate process)

Docker would be great for portability / modularity but not essential.